gulpfile.js

```diff
@@ -12,8 +15,58 @@ require('laravel-elixir-vue-2');
  | file for our application, as well as publishing vendor resources.
  |
  */
+elixir.extend('templates', function () {
+    new Task('templates', function () {
+        return gulp.src('./resources/angular/**/*.html')
+            .pipe(templateCache())
```

Comment:
What we have at work looks like it adds a couple parameters to this.  The first is the output file, 'template.js', the second is a config object, {standalone: true, module: 'templates.app'}.  The module lets it get referenced by the angular code (see the comment in app.js that I will make shortly).

Try making it like this: 
.pipe(templateCache('templates.js', {standalone: true, module: 'templates.app'}))